src/main.c

```diff
@@ -75,9 +96,9 @@ static const uint8_t CLA = 0xD7;
 // and sent in the next io_exchange call.
 static void cardano_main(void)
 {
-	volatile unsigned int rx = 0;
-	volatile unsigned int tx = 0;
-	volatile unsigned int flags = 0;
+	volatile size_t rx = 0;
```

Comment:
Hmm, what is this diffing against? I presumed these changes were already merged with conversion fixes

Reply:
It's diffing against master but I can make it diff against your wip_nano_x branch, though this may hide some of the code that I did not change and I think it makes sense to review the nano x support as a whole